plutus-pab/src/Plutus/PAB/Types.hs

```diff
@@ -123,9 +159,10 @@ data WebserverConfig =
 defaultWebServerConfig :: WebserverConfig
 defaultWebServerConfig =
   WebserverConfig
-    { baseUrl              = BaseUrl Http "localhost" 8080 "/"
+    { baseUrl              = BaseUrl Http "localhost" 9080 ""
```

Comment:
I changed this to what I've seen in the yaml configs. This may affect people who have hard-coded 8080 in some of their curl scripts, or similar, and somehow depended on this default.